assigner/baserepo.py

```diff
@@ -383,13 +383,14 @@ def new(cls, name, namespace, config):
                 len(namespaces), namespace, namespaces[0]["path"]
             )
             logging.warning(
-                "(please update the configuration in your namespace to " +
+                "(please update the configuration in your namespace to %s",
```

Comment:
That's some weird wrapping. This should work:

```suggestion
                "(please update the configuration in your namespace to "
```